core/config/project_settings.h

```diff
@@ -243,3 +250,27 @@ Variant _GLOBAL_DEF(const PropertyInfo &p_info, const Variant &p_default, bool p
 #define GLOBAL_DEF_RST_NOVAL_BASIC(m_var, m_value) _GLOBAL_DEF(m_var, m_value, true, true, true)
 
 #define GLOBAL_DEF_INTERNAL(m_var, m_value) _GLOBAL_DEF(m_var, m_value, false, false, false, true)
+
+/////////////////////////////////////////////////////////////////////////////////////////
+// Cached versions of GLOBAL_GET.
+// Cached but uses a typed variable for storage, this can be more efficient.
+// N.B. Variables are prefixed with _ggc_ because compiler is set to report
+// shadowed variable names outside the lambda.
```

Comment:
I don't think this needs to be documented

Reply:
That's because you know what it's for. :wink:

Reply:
Ok, having the comment works for me too :)

Reply:
Will change that, your version sounds better. :+1: